multiqc/utils/config_defaults.yaml

```diff
@@ -160,6 +160,7 @@ fn_clean_exts:
     - '_peaks.xls'
     - '.relatedness'
     - '.cnt'
+    - '.stats'
```

Comment:
This is a little generic perhaps, and I'm worried that it could appear in the middle of genuine filenames and cause incorrect truncation. Could it be moved to `fn_clean_trim` instead so that it's only removed if it's at the end of the filename?

Reply:
Yup, should happen with both as long as `.txt` is first in the list.